src/main/java/kostka/moviecatalog/service/runtimeconfiguration/RuntimeConfigurationService.java

```diff
@@ -13,36 +13,37 @@
 
 import javax.validation.Valid;
 import java.util.Map;
+import java.util.Optional;
 
 import static kostka.moviecatalog.factory.RuntimeConfigurationFactory.getOptionsClass;
 
 @Service
 public class RuntimeConfigurationService {
     private static final Logger LOGGER = LoggerFactory.getLogger(RuntimeConfigurationService.class);
-    private RuntimeConfigRepository runtimeConfigRepository;
+    private final RuntimeConfigRepository runtimeConfigRepository;
 
-    private ObjectMapper mapper = new ObjectMapper();
+    private final ObjectMapper mapper = new ObjectMapper();
 
     @Autowired
     public RuntimeConfigurationService(final RuntimeConfigRepository runtimeConfigRepository) {
         this.runtimeConfigRepository = runtimeConfigRepository;
     }
 
-    public RuntimeConfiguration updateRuntimeConfiguration(
+    public RuntimeConfiguration update(
             final RuntimeConfigDto dto) {
-        String configName = dto.getConfigName();
+        RuntimeConfigurationEnum runtimeConfigEnum = getRuntimeEnumByName(dto.getConfigName());
```

Comment:
nit: klidně bych tu proměnnou pojmenoval i jako runtimeConfigType.

Mít datový typ v názvu proměnné, někdy není úplně fajn. Na to doporučuju najít si, co byla Hungarian notation :) V původním WinAPI, to byl luxus používat.

Reply:
rozkaz sefe :) dava to smysl ;)